docs/resources/post_process_aws_connection.md

```diff
@@ -0,0 +1,60 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "clumio_post_process_aws_connection Resource - terraform-provider-clumio"
+subcategory: ""
+description: |-
+  Post process Clumio AWS Connection Resource used to post-process AWS connection to Clumio.
+---
+
+# clumio_post_process_aws_connection (Resource)
+
+Post process Clumio AWS Connection Resource used to post-process AWS connection to Clumio.
+
+## Example Usage
+
+```terraform
+resource "clumio_post_process_aws_connection" "example" {
+  # example configuration here
+  token                              = "mytoken"
+  role_external_id                   = "role_external_id"
+  account_id                         = "account_id"
+  region                             = "region"
+  role_arn                           = "role_arn"
+  config_version                     = "1"
+  discover_version                   = "3"
+  protect_config_version             = "18"
+  protect_ebs_version                = "19"
+  protect_rds_version                = "18"
+  protect_ec2_mssql_version          = "1"
+  protect_warm_tier_version          = "2"
+  protect_warm_tier_dynamodb_version = "2"
+  protect_s3_version                 = "1"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **account_id** (String) The AWS Customer Account ID.
+- **config_version** (String) Clumio Config version.
+- **discover_version** (String) Clumio Discover version.
+- **region** (String) The AWS Region.
+- **role_arn** (String) Clumio IAM Role Arn.
+- **role_external_id** (String) A key that must be used by Clumio to assume the service role in your account. This should be a secure string, like a password, but it does not need to be remembered (random characters are best).
+- **token** (String) The AWS integration ID token.
+
+### Optional
+
+- **id** (String) The ID of this resource.
+- **protect_config_version** (String) Clumio Protect Config version.
+- **protect_dynamodb_version** (String) Clumio DynamoDB Protect version.
+- **protect_ebs_version** (String) Clumio EBS Protect version.
+- **protect_ec2_mssql_version** (String) Clumio EC2 MSSQL Protect version.
+- **protect_rds_version** (String) Clumio RDS Protect version.
+- **protect_s3_version** (String) Clumio S3 Protect version.
+- **protect_warm_tier_dynamodb_version** (String) Clumio DynamoDB Warmtier Protect version.
+- **protect_warm_tier_version** (String) Clumio Warmtier Protect version.
```

Comment:
`Warmtier` -> `Warm Tier`?
Also what's the difference behind the `DynamoDB` Warm Tier version (line 57) and just the `Warm Tier`version (line 58)?

Reply:
`"protect": {
	"warmTier": {
		"config": {
			"version": "1",
		}
		"dynamodb": {
			"version": "1",
		}
	}
}`
protect_warm_tier_version is the version inside protect->warmTier->config.
protect_warm_tier_dynamodb_version is the version inside protect->warmTier->dynamodb.